app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :username, presence: true, uniqueness: { case_sensitive: false }
+  validates :email, presence: true
```

Comment:
- [ ] [OPTIONAL] Use Regular expressions or Rails built-in methods to check if the string in the email field is indeed formatted as a vallid email addess.